packages/actor-rdf-resolve-hypermedia-links-queue-priority/lib/ActorRdfResolveHypermediaLinksQueuePriority.ts

```diff
@@ -0,0 +1,27 @@
+import type {
+  IActionRdfResolveHypermediaLinksQueue,
+  IActorRdfResolveHypermediaLinksQueueArgs,
+  IActorRdfResolveHypermediaLinksQueueOutput,
+} from '@comunica/bus-rdf-resolve-hypermedia-links-queue';
+import { ActorRdfResolveHypermediaLinksQueue } from '@comunica/bus-rdf-resolve-hypermedia-links-queue';
+import type { IActorTest } from '@comunica/core';
+import { LinkQueuePriority } from './LinkQueuePriority';
+
+/**
+ * A comunica Priority RDF Resolve Hypermedia Links Queue Actor.
+ */
+export class ActorRdfResolveHypermediaLinksQueuePriority extends ActorRdfResolveHypermediaLinksQueue {
+  public constructor(args: IActorRdfResolveHypermediaLinksQueueArgs) {
+    super(args);
+  }
+
+  // eslint-disable-next-line  unused-imports/no-unused-vars
```

Comment:
Everywhere you ignore this rule, you can just prefix the variable name with `_`.